build/images/ovs/Dockerfile.windows

```diff
@@ -14,34 +14,37 @@
 
 ARG NANOSERVER_VERSION
 
-FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as ovs-base
 
 ARG WIN_OVS_VERSION
 
-# Switch to a privileged user
-USER ContainerAdministrator
```

Comment:
May I know why we previously switched to the privileged user? cc @rajnkamr

Reply:
From my test, the privileged user is needed if we expect to access system path, e.g. installing/copying files into `C:\windows\system32` or  path `c:\`, this is observed when running inside nanoserver images. But for servercore image, it looks the default user already has super privileges to install files or copy files, at least I didn't hit errors if I remove those lines in the test. But for the final image with nanoserver, the privileges are necessary to copy files or access files in system paths.

Reply:
Got it, thanks.

Reply:
In order to set system PATH, ContainerAdministrator should be used,  like for -  "RUN setx /m PATH ", it was used for same reason,
For copying from servercore to nanoserver layer, system path could work with USER ContainerUser  in shell type powershell, 
However when switched to shell type 'cmd', it might need User ContainerAdministrator for copying from system path !